### PR TITLE
Remove DeprecatedFileStream package

### PR DESCRIPTION
**Pull request by guillep**
Finish the cleaning and removing the DecatedFileStream package that is deprecated since a couple of years already.

### CODE REVIEW

**Comment by astares** [comment]
Just to make sure: where Zinc changes applied in @svenvc repo already so we do not have a problem when next Zinc version is merged?

**Comment by guillep** [comment]
Let me check, but I this PR does remove old streams, reusing the ZnStreams but it should not change anything in Zinc.
Did you see something in particular?

File: src/Zinc-Zodiac/ZnHTTPSTest.class.st [hunk 1]
```diff
@@ -36,7 +36,7 @@ ZnHTTPSTest class >> generateTestFiles [
 	
 	(Integer primesUpTo: 100) do: [ :each | | size |
 		size := 1024 * each + each.
-		FileStream fileNamed: ('test-', size asString, '.txt') do: [ :stream |
+		('test-', size asString, '.txt') asFileReference writeStreamDo: [ :stream |
```
Comment (guillep) [inline src/Zinc-Zodiac/ZnHTTPSTest.class.st:39]: There is this test

File: src/Fuel-Core/ZnBufferedReadStream.extension.st [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Extension { #name : #ZnBufferedReadStream }
+
+{ #category : #'*Fuel-Core' }
+ZnBufferedReadStream >> fuelNextWordsInto: aWordObject [
```
Comment (guillep) [inline src/Fuel-Core/ZnBufferedReadStream.extension.st:4]: And this extension methods from Fuel

**Review by guillep — COMMENTED** [review]
I don't think the extension method needs to be pushed, but maybe the test does.

@svenvc how would you like to proceed?

**Review by astares — APPROVED** [review]

**Comment by astares** [comment]
Wrong window - please remove the approval

**Comment by guillep** [comment]
Tests are green! https://ci.inria.fr/pharo-ci-jenkins2/blue/organizations/jenkins/Test%20pending%20pull%20request%20and%20branch%20Pipeline/detail/PR-7885/8/tests

<blockquote><div><strong><a href="https://ci.inria.fr/pharo-ci-jenkins2/blue/organizations/jenkins/Test%20pending%20pull%20request%20and%20branch%20Pipeline/detail/PR-7885/8/tests">Jenkins</a></strong></div></blockquote>

**Pull request merged**